src/components/product/ProductCard/ProductCard.tsx

```diff
@@ -177,7 +194,9 @@ export const fragment = graphql`
 
     offers {
       lowPrice
+      offerCount
```

Comment:
Assuming we went with `availability`, do we still need to include this `offerCount`?